lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_ansible_method_base.rb

```diff
@@ -80,6 +83,13 @@ def update_object_with_stats(key, value)
       object.object_send(:save!)
     end
 
+    def service_var_with_stats(key, value)
+      object = @workspace.current['service'] || @workspace.root['service']
+      raise _("Object update failed - service object not found for: [service_var__%{key} = %{value}]") % {:key => key, :value => value} if object.blank?
```

Comment:
@lfu would it make sense to write a small helper function that did the normalization of variables names that adheres to the Ansible variable name restrictions.